pkg/commands/git_commands/archive.go

```diff
@@ -0,0 +1,43 @@
+package git_commands
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/utils"
+)
+
+type ArchiveCommands struct {
+	*GitCommon
+}
+
+func NewArchiveCommands(gitCommon *GitCommon) *ArchiveCommands {
+	return &ArchiveCommands{
+		GitCommon: gitCommon,
+	}
+}
+
+func (self *ArchiveCommands) Archive(refName string, archiveName string, prefix string) error {
+	cmdArgs := NewGitCmd("archive").ArgIf(prefix != "", "--prefix", prefix).Arg("-o", archiveName, refName)
+
+	return self.cmd.New(cmdArgs.ToArgv()).DontLog().Run()
```

Comment:
Why `DontLog()`? I think it's nice to see the command in the log.

And call `LogAction` before, but probably not from here, but from the controller (or helper), I'm too lazy to look up how we usually do this.